libbeat/docs/upgrading.asciidoc

```diff
@@ -1,160 +1,329 @@
+----
+
+To check the template version, open the {kib} Console and run:
 
-To check which version of the template is loaded, open Kibana Console, call `GET
-/_template/metricbeat`, and look for the version string. Note that you need to
-do this for each Beat type that you are running (e.g. Filebeat, Metricbeat,
-Packetbeat).
+[source,js]
+----
+GET /_template/metricbeat
+----
+// CONSOLE
 
+Look for the version string in the response. Repeat this step for every Beat
+running in your environment.
+
+*Metricbeat and Filebeat users:* If you use {beats} central management,
+make sure you migrate the {beats} central management index to 6.7 before you
+upgrade to 7.0. Although central management is not a GA-level feature in 7.0,
+we've provided a migration tool to help you migrate your configurations. For
+more information, see the
+https://www.elastic.co/blog/beats-6-7-0-released[Beats 6.7.0 release blog].
+
+[[migrate-config-files]]
 ==== Migrate configuration files
 
-Beats 6.0 comes with several backwards incompatible configuration changes.
-Please review the <<breaking-changes-6.0>> document. Where possible, we kept the
-old configuration options working, but deprecated them.  However, deprecation
-was not always possible, so if you use any of the settings mentioned in the
-Breaking Changes section of the release notes, make sure you understand the
+{beats} 7.0 comes with several backwards incompatible configuration changes.
+Before upgrading, review the <<breaking-changes-7.0>> document. Also review
+the full list of breaking changes in the <<release-notes>> for 7.0.
+
+Where possible, we kept the old configuration options working, but deprecated
+them. However, deprecation was not always possible, so if you use any of the
+settings described under breaking changes, make sure you understand the
 alternatives that we provide.
 
-===== modules.d configuration layout
+[[enable-ecs-compatibility]]
+==== Enable the compatibility layer for Elastic Common Schema (ECS) fields
 
-Starting with the 6.0 version, Filebeat and Metricbeat are moving to a directory
-layout for configuration, where each module is configured in its own
-configuration file.
+Starting with 7.0, the fields exported by {beats} conform to the
+{ecs-ref}/index.html[Elastic Common Schema (ECS)]. Many of the exported fields
+have been renamed. See {beats-ref}/breaking-changes-7.0.html[Breaking
+changes in 7.0] for the full list of changed names.
 
-While the all-in-one configuration is still fully supported, we recommend moving
-to the new layout at upgrade time. This typically means starting off with the new
-default configuration and modifying it with the custom settings that you had in
-your old files.
+To help you transition to the new fields, we provide a compatibility layer in
+the form of ECS-compatible field aliases. To use the aliases, set
+`migration.6_to_7.enabled: true` in the Beat's configuration file *before* you
+run {beats} 7.0 to ingest data for the first time.
```

Comment:
This setting must actually be set before running the beat setup, as it creates the field aliases in the index template

Reply:
Yup...originally I assumed users would use lazy loading of the index template and run the beat at the same time, but with the explicit template loading, this sentence is def misleading.